### ISSUE TITLE
Screen Reader section

### ISSUE THREAD

**Issue by AndrewArch** [govau/content-guide#97]
Screen Reader section 

## What do you need?

Screen Reader section should be a subset of Assistive Technologies section (additional description of Screen magnifiers, Speech recognition, etc can then be added

## Which entry/section is this related to?

Assistive Technologies & Screen Redaers

## Good example

NA

**Comment by gazanson** [comment]
Now being tracked in Trello
